qa/os/src/test/java/org/elasticsearch/packaging/test/ArchiveTests.java

```diff
@@ -413,4 +408,11 @@ public void test94ElasticsearchNodeExecuteCliNotEsHomeWorkDir() throws Exception
         }
     }
 
+    @After
+    public void tearDown() throws Exception {
+        // clean up logs between logs so we only show logs from the running tests if we dump them on failure
+        FileUtils.rm(installation.logs);
```

Comment:
Would it make sense to move this to the parent class (`PackagingTestCase`) so that avoid having our logs carry over for all test groupings?